internal/charm/styles/styles.go

```diff
@@ -133,6 +135,33 @@ func MakeBoxed(s string, borderColor lipgloss.AdaptiveColor, alignment lipgloss.
 		Render(s)
 }
 
+// MakeSection returns a string enclosed in a top and bottom border with a title
+func MakeSection(title, content string, color lipgloss.AdaptiveColor) string {
```

Comment:
The point of this change it to make things more copy-pastable by just having top and bottom lines instead of a side border